examples/simple_experiment.rs

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2019 Gabriela Alexandra Moldovan created by the Software Development Team
+// Copyright (c) 2019 Gabriela Alexandra Moldovan
+// Copyright (c) 2019 King's College London created by the Software Development Team
```

Comment:
Ah, this isn't your fault, but  the wording sounds like King's was created by our team. Laurie had the same issue in one of his repos recently. I guess the same one you lifted the license from.

How about:
```
// Copyright (c) 2019 Gabriela Alexandra Moldovan
// Copyright (c) 2019 King's College London (The Software Development Team)
```

I hesitate to use `created by the Software Development Team` because I feel it undermines Gabi's contribution. She created it more than anyone else at this point :)

[@ltratt should sign this one off, as he's the license guru around here]

Reply:
I updated the license as you suggested, but I am happy with either version :).